t/t.go

```diff
@@ -802,11 +807,16 @@ func downloadDataFiles() {
 		if out, err := cmd.CombinedOutput(); err != nil {
 			fmt.Printf("Error %v", err)
 			fmt.Printf("Output %v", out)
+			outInfo = append(outInfo, out)
+			errInfo = append(errInfo, err)
 		}
 	}
+	return outInfo, errInfo
```

Comment:
can we check for error right here instead of returning it?

Reply:
Resolved